src/document/DocumentAnnotator.ts

```diff
@@ -63,6 +63,8 @@ export default class DocumentAnnotator extends BaseAnnotator {
         this.annotatedEl.classList.add(CLASS_ANNOTATIONS_LOADED);
 
         this.render();
+
+        this.setLoaded();
```

Comment:
Should this happen in the BaseAnnotator's `init` method?

Reply:
I did it here to ensure the  `annotatedEl` was ready to be scrolled to.